Sources/KnitCodeGen/FunctionCallRegistrationParsing.swift

```diff
@@ -111,7 +112,8 @@ extension FunctionCallExprSyntax {
                 concurrencyModifier: concurrencyModifier,
                 registrationArguments: registrationArguments,
                 leadingTrivia: leadingTrivia,
-                functionName: .implements
+                functionName: .implements,
+                syntax: implementsCalledMethod.calledExpression
```

Comment:
![Screenshot 2025-02-05 at 2 22 44 PM](https://github.com/user-attachments/assets/a42e63e3-5c34-4a00-b66c-5e930f9502c0)

This is placing the error on the wrong syntax. Can we use
```suggestion
                syntax: implementsCalledMethod.arguments
```

Reply:
You can merge this as is, i have a follow up to place the errors on the comment lines

Reply:
https://github.com/cashapp/knit/pull/242/